tests/assets/deployment.manifest_invalid.json

```diff
@@ -25,15 +25,15 @@
             "edgeAgent": {
               "type": "docker",
               "settings": {
-                "image": "mcr.microsoft.com/azureiotedge-agent:1.0"
+                "image": "mcr.microsoft.com/azureiotedge-agent:${RUNTIME_TAG}"
```

Comment:
>${RUNTIME_TAG} [](start = 63, length = 14)

Is the manifest invalid because the env variable isn't sepcified?

Reply:
I believe this file is being used to test against a user generated manifest that is invalid somewhere, not because of the .env variable.